ArkKit/ark-render-kit/canvas-components/input/ButtonCanvasComponent.swift

```diff
@@ -3,25 +3,23 @@ import CoreGraphics
 struct ButtonCanvasComponent: AbstractTappable, CanvasComponent {
     let width: Double
     let height: Double
-    let center: CGPoint
-    let rotation: Double
+    var center: CGPoint
+    var rotation: Double
     let areValuesEqual: AreValuesEqualDelegate
 
     var onTapDelegate: TapDelegate?
 
-    init(width: Double, height: Double, center: CGPoint, rotation: Double = 0.0,
+    init(width: Double, height: Double, rotation: Double = 0.0,
```

Comment:
maybe we can instead do an optional `center = CGPoint.zero` in the constructor?

since rotation also has an optional param in the constructor

Reply:
decided on removing and only setting via `center` and `rotation`